bindata/network/openshift-sdn/sdn.yaml

```diff
@@ -235,6 +235,62 @@ spec:
         - name: sdn-metrics-certs
           mountPath: /etc/pki/tls/metrics-certs
           readOnly: True
+      {{- if .SDNPlatformAzure}}
+      - name: drop-icmp
+        image: {{.SDNImage}}
+        command:
+        - /bin/bash
+        - -c
+        - |
+          set -xe
+
+          cat <<'EOF' > /var/run/add_iptables.sh
+          #!/bin/sh
+          if [ -z "$3" ]
+          then
+               echo "Called with host address missing, ignore"
+               exit 0
+          fi
+          echo "Adding ICMP drop rule for '$3' "
+          if iptables -C CHECK_ICMP_SOURCE -p icmp -s $3 -j ICMP_ACTION
+          then
+               echo "iptables already set for $3"
+          else
+               iptables -A CHECK_ICMP_SOURCE -p icmp -s $3 -j ICMP_ACTION
+          fi
+          EOF
+
+          chmod +x /var/run/add_iptables.sh
```

Comment:
So if we're mounting `host-slash` on `/` then this is the root `/var/run` right? so we should be a bit more paranoid and `chmod` it to a specific value (0755?). Actually, maybe do

    touch /var/run/add_iptables.sh
    chmod 0755 /var/run/add_iptables.sh
    cat <<'EOF' > /var/run/add_iptables.sh
    ...

or else

    mkdir -m 0700 /var/run/drop-icmp
    cat <<'EOF' > /var/run/add_iptables.sh
    ...